ushahidi/platform-client
# Fixes to improved categories

This pull request makes the following changes:
- Fix styling & markup in categories editor
- Don't allow making 'tags' attributes private

Test these changes by:
- View category editor and check it matches PL
- Edit a categories field and check it can't be made private

Refs ushahidi/platform#1397

Ping @ushahidi/platform


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/ushahidi/platform-client/649)
